### PR TITLE
Correct docs moving and add example from source code

### PR DESCRIPTION
**Pull request by zhongjiajie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,14 +40,14 @@ nix profile install github:laixintao/flameshow
 View golang's goroutine dump:
 
 ```shell
-$ curl http://localhost:9100/debug/pprof/goroutine -o goroutine.out
+$ curl https://raw.githubusercontent.com/laixintao/flameshow/main/tests/pprof_data/goroutine.out -o goroutine.out
```
[the remark this answers is not in the document]
Comment (laixintao) [reply README.md:43]: I prefer not changing this line.

The code is for showing user how to view golang's pprof data. User will understand `/debug/pprof/goroutine` is a pprof format.

**Comment by laixintao** [comment]
Thank you

File: README.md (hunk 1 above)
Comment (zhongjiajie) [inline README.md:43]:
```suggestion
$ curl http://localhost:9100/debug/pprof/goroutine -o goroutine.out
```

[the remark this answers is not in the document]
Comment (zhongjiajie) [reply README.md]: Got it and revert the change

**Comment by zhongjiajie** [comment]
BTW, ci fail seems we do not pin black version, should I approve the change for latest version of black in this PR, or something else?

**Comment by laixintao** [comment]
merged, thanks.

And yes, black should be pinned.

**Pull request merged**